karma.conf.js

```diff
@@ -1,71 +1,57 @@
 // Karma configuration
 // Generated on Wed Jun 18 2014 07:15:37 GMT+0100 (GMT Summer Time)
 
-module.exports = function(config) {
+module.exports = function (config) {
```

Comment:
This file looks unchanged other than formatting, may not need to include these changes?